BootloaderCorePkg/Stage2/Stage2Support.c

```diff
@@ -637,6 +635,18 @@ BuildExtraInfoHob (
     PlatformUpdateHobInfo (&gLoaderPlatformInfoGuid, LoaderPlatformInfo);
   }
 
+  // Build Performance Hob
```

Comment:
Different topic - this can be in a separate PR.

Reply:
OK. will change it in another PR.